hepcrawl/loaders.py

```diff
@@ -99,3 +101,13 @@ class HEPLoader(ItemLoader):
     dois_out = ListToValueDict()
     related_article_doi_out = ListToValueDict()
     urls_out = ListToValueDict(key="url")
+
+
+class ElsevierLoader(HEPLoader):
```

Comment:
Do you think we shall have the MathML cleaning for all records perhaps? So that you just need to amend the existing `abstract_in` loader?